apps/i18n/common/en_us.json

```diff
@@ -12,6 +12,10 @@
   "answersVisible": "Answers visible (read-only)",
   "assessmentSteps": "Steps to give assessment for",
   "assessmentSettings": "Assessment Settings",
+  "assign": "Assign",
+  "assignACourse": "Assign a course to your classroom or start your own course.",
+  "assignCourse": "Assign Course",
+  "assignCourseConfirm": "Are you sure you want to assign \"{courseName}\" to \"{sectionName}\"?",
```

Comment:
Since a section can only have one course assigned at a time, it'd be nice if this dialog mentioned it when you are replacing a course assignment (maybe even which course you're replacing).  Likewise, do we want special handling for the no-op case (assigning a section's current course through this flow)?